content/guides/comparison.md

```diff
@@ -1,12 +1,17 @@
 ---
-title: Bundler Comparison
+title: Comparison
 sort: 25
 contributors:
   - pksjce
   - bebraw
   - chrisVillanueva
   - tashian
   - simon04
+related:
+  - title: JSPM vs. Webpack
```

Comment:
We mostly also use lowercase "webpack" for external page titles, e.g. see d191256b0ff58a51104f47cf377b51a90ce518a4.